src/WebJobs.Script/DependencyInjection/ScriptStartupTypeLocator.cs

```diff
@@ -269,8 +272,12 @@ private void ValidateBundleRequirements(ExtensionBundleDetails bundleDetails)
             }
         }
 
-        private void ValidateExtensionRequirements(List<Type> startupTypes)
+        private void ValidateExtensionRequirements(List<Type> startupTypes, ExtensionRequirementsInfo requirementsInfo)
         {
+            if (requirementsInfo.ExtensionRequirementsByStartupType == null)
```

Comment:
nit: switching to the new `is` and `is not` syntax across the board

Reply:
done